tests/services/inventory/test_data/mock_gcp_results.py

```diff
@@ -274,11 +274,36 @@
         },
         "id": "project3:dataset1",
         "kind": "bigquery#dataset"
+    }],
+    # CAI uses project ID instead of number.
+    "project3": [{
+        "datasetReference": {
+            "datasetId": "dataset1",
+            "projectId": "project3"
+        },
+        "id": "project3:dataset1",
+        "kind": "bigquery#dataset"
     }]
 }
 
 BQ_GET_DATASET_ACCESS = {
-    PROJECT_ID_PREFIX + "3": {
+     PROJECT_ID_PREFIX + "3": {
```

Comment:
Same here, this is raw API access so it needs to use the project number since that's how we call the API. CAI is not involved.

Reply:
Updated.